### ISSUE TITLE
About VQA mlm loss

### ISSUE THREAD

**Issue by katrina433** [salesforce/ALBEF#90]
Hello! 

I have a question about the mlm loss computation in [model_vqa.py](https://github.com/salesforce/ALBEF/blob/main/models/model_vqa.py). It seems like `alpha` is never used in `forward`, so the [`answer_output`](https://github.com/salesforce/ALBEF/blob/main/models/model_vqa.py#L97) is independent of `logits_m`. Is this the intended behavior?

**Comment by LiJunnan1992** [comment]
Hi, thanks for pointing this out. The code has been updated now to use alpha when self.distill=True.
